blog/Dockerfile

```diff
@@ -1,30 +1,14 @@
-FROM node:8-alpine
+FROM node:12
 
-RUN addgroup -S mol && adduser -S -g mol mol
+RUN mkdir /app
+WORKDIR /app
 
-ENV HOME=/home/mol
-ENV HOME_APP=$HOME/app
-ENV NODE_ENV=production 
-
-WORKDIR $HOME_APP
+ENV NODE_ENV=production
 
 COPY package.json .
```

Comment:
Please use `COPY package*.json ./`

Reply:
Done in https://github.com/moleculerjs/moleculer-examples/pull/22/commits/d226aa09a9302e9dd27e55c4f6291aa5fc555a22
I've also made the same change in `conduit/Dockerfile`